app/models/ability.rb

```diff
@@ -166,6 +166,10 @@ def signed_in_with_organizer_role(user)
     can [:edit, :update, :toggle_user], Role do |role|
       role.resource_type == 'Conference' && (conf_ids_for_organizer.include? role.resource_id)
     end
+
+    can [:index, :revert], PaperTrail::Version do |version|
```

Comment:
We should properly authorize `revert` action. Please read my latest comments on the PR about this. Then you disable the revert button based on authorization and not custom checks. Same for the revert buttons inside the table with the changes.